### PR TITLE
Make the commit message fit better in various renderings.

### PR DESCRIPTION
**Pull request by nedbat**
- Short subject line.

- More details in the body after a blank line.

### CODE REVIEW

**Comment by nedbat** [comment]
@timmc-edx Look ok?

File: jenkins/cleanup_python_code.py [hunk 1]
```diff
@@ -44,14 +44,21 @@
...
+        "\n" +
+        "The following packages were installed:\n" +
+        "{1}"
+    ).format(scripts, packages)
 
     pull_request_creator = PullRequestCreator(sha=sha, repo_root=repo_root, branch_name=CODE_CLEANUP_BRANCH_NAME,
                                               user_reviewers=user_reviewers, team_reviewers=team_reviewers,
-                                              commit_message=pr_message, pr_title="Python Code Cleanup",
+                                              commit_message=commit_message, pr_title="Python Code Cleanup",
                                               pr_body=pr_message)
```
Comment (mraarif) [inline jenkins/cleanup_python_code.py:62]: maybe we should also pass the same commit message for `pr_body` as well, since `pr_message` no longer exists?

Comment (nedbat) [reply jenkins/cleanup_python_code.py:62]: oops :)

Comment (nedbat) [reply jenkins/cleanup_python_code.py]: Fixed.

**Review by mraarif — APPROVED** [review]

**Pull request merged**